hadoop-ozone/s3gateway/pom.xml

```diff
@@ -78,6 +78,10 @@
       <groupId>com.fasterxml.jackson.dataformat</groupId>
       <artifactId>jackson-dataformat-xml</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-jaxb-annotations</artifactId>
```

Comment:
just curious why moving up to 2.13 led to addition of this extra dependency?

Reply:
From [release notes](https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.13#compatibility-module-changes):

> `jackson-dataformat-xml` no longer has direct dependency on `jackson-module-jaxb-annotations`

So we need to explicitly add it.